serverless_llm/serve/schedulers/fcfs_scheduler.py

```diff
@@ -20,16 +20,15 @@
 import time
 from typing import Mapping, Optional
 
-import ray
-
 from serverless_llm.serve.logger import init_logger
-from serverless_llm.serve.schedulers.scheduler_utils import SllmScheduler
 from serverless_llm.serve.utils import get_worker_nodes
 
+from .scheduler_utils import SllmScheduler
+
 logger = init_logger(__name__)
 
 
-@ray.remote(num_cpus=1, resources={"control_node": 0.1})
+# @ray.remote(num_cpus=1, resources={"control_node": 0.1})
```

Comment:
If we move the ray into runtime, perhaps we could delete the decorator instead of only comment it out?

Reply:
ok, cleaned up.